### ISSUE TITLE
Error: could not list clusters: failed to list nodes: exit status 1

### ISSUE THREAD

**Issue by oneslideicywater** [kubernetes-sigs/kind#864]
I create a VM to test kind with network setted by NAT.

after a VM restart,I notice a ip changed due to DHCP,and I get this error when I
type `kind create cluster`  in command line

**Comment by BenTheElder** [comment]
Can you give us a little more info about the environment?
If you increase the log level we can see what errored as well.

Also note that kind clusters do not survive host restarts currently, there's another issue tracking that problem.

**Comment by tao12345666333** [comment]
> after a VM restart

Has your Docker started?

**Comment by oneslideicywater** [comment]
@tahsinrahman yes,Docker isn't Runing,but @BenTheElder the cluster is not function properly,when I
restart docker and test the cluster with `kubectl cluster-info`,I have to delete the cluster and recreate another. by the way, how can increase the log level ?

**Comment by tao12345666333** [comment]
> the cluster is not function properly,when I
> restart docker and test the cluster with `kubectl cluster-info`,I have to delete the cluster and recreate another.

About cluster restart, there are already additional issues for tracking. #148 



> how can increase the log level ?

`--loglevel string   logrus log level [panic, fatal, error, warning, info, debug, trace] (default "warning")`
